site/powered_by.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+## Powered By
+
+### Project and Product names using "Apache Arrow"
+
+Organizations creating products and projects for use with Apache Arrow, along
```

Comment:
I used this paragraph from Spark but extended the wording to `Apache Arrow`. Spark was also talking about `Spark` alone but from my understanding they have a "more explicit trademark" on that then Arrow has.